protocol/protos/NetRemoteWifi.proto

```diff
@@ -35,3 +35,19 @@ message WifiConfigureAccessPointResult
     //
     bool Succeeded = 2;
 }
+
+message WifiEnumerateAccessPointsResultItem
+{
+    string AccessPointId = 1;
+    Microsoft.Net.Wifi.AccessPointCapabilities Capabilities = 2;
+    bool IsEnabled = 3;
+}
+
+message WifiEnumerateAccessPointsRequest
+{
+}
+
+message WifiEnumerateAccessPointsResult
+{
+    repeated Microsoft.Net.Remote.Wifi.WifiEnumerateAccessPointsResultItem AccessPoints = 1;
```

Comment:
FYI @corbin-phipps, I opted to use a list of `WifiEnumerateAccessPointsResultItem`s which includes the `AccessPointId` instead of `map<string, WifiEnumerateAccessPointsResultItem>` because the client would presumably never actually do a lookup on the map (which is what maps/dictionaries are designed for), but instead would immediately just iterate or filter it for the properties it wants, which is inherently more list-like.

This then raises the question of whether we should add something to `WifiEnumerateAccessPointsRequest` which allows filtering the results with the condition(s) it's looking for. Given we expect this list to be relatively small (on the order of 10s at absolute _worst_), it didn't seem appropriate to complicate the API and implementation with filtering options.

What do you think?

Reply:
I think leaving the filtering to the client is fine. In general, I think simpler APIs with fewer options tend to be best and makes them easier to maintain